x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_snooze/panel/base_snooze_panel.tsx

```diff
@@ -223,7 +223,7 @@ export const BaseSnoozePanel: React.FunctionComponent<BaseSnoozePanelProps> = ({
                           height: '36px',
                           // Replicate euiPanel--accent vs euiPanel--subdued
                           // Applying these classNames by themselves doesn't work due to a CSS-in-JS issue with EuiPanel
-                          color: isActive ? '#a8376a' : euiTheme.colors.subduedText,
+                          color: isActive ? '#a8376a' : euiTheme.colors.textSubdued,
```

Comment:
We should be able to use a token for the static colors as well, I think?
`#a8376a` could become `euiTheme.colors.textAccent` which is `#A11262` (light) and `#EE72A6` (dark).

Reply:
i ll update it, thanks!